docs/source/azure/examples/addstoragefirewall.rst

```diff
@@ -0,0 +1,28 @@
+Add storage firewall
```

Comment:
Please update `cloud-custodian/docs/source/azure/policy/resources/storage.rst` with new action and describe possible parameters. (Probably worth to mention that if parameter is not specified, it remains unchanged).

You can also insert links to these examples to the storage documentation page.